src/main/java/com/akathist/maven/plugins/launch4j/generators/Launch4jFileVersionGenerator.java

```diff
@@ -0,0 +1,69 @@
+package com.akathist.maven.plugins.launch4j.generators;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class Launch4jFileVersionGenerator {
```

Comment:
Wouldn't be good to add a private constructor as this is pure utility class?

Reply:
Of course, it would be good ;-) I've added it for the second pure utility class as well.